chore: debug server for local testing (#159)
adobe/helix-sidekick: README.md

#### Test a local Sidekick version

> 1. Run `npm run debug` on your local checkout of this repository

Reviewer: ```suggestion
1. Run `npm start` on your local checkout of this repository
```

2. Install a Sidekick bookmarklet for your project: https://www.hlx.live/tools/sidekick/
3. Edit the bookmarklet URL by replacing `https://www.live/tools/sidekick/app.js` with `http://localhost:3000/src/sidekick/app.js`
4. Click the bookmarklet to launch your local version

#### Test a local project config

If you want to test a [config](#configuration) file before deploying it to your project:

> 1. Run `npm start` on your local checkout of this repository

Reviewer: wouldn't it just be:

```suggestion
1. Run `hlx up` on your local checkout of the project repository
```

> 1. Run `npm run debug` on your local checkout of the project repository

Reviewer: ```suggestion
1. Run `npm start` on your local checkout of the project repository
```
